include/sparrow/data_type.hpp

```diff
@@ -117,6 +117,7 @@ namespace sparrow
                         // the right type?
         ,
         bool,
+        char,
```

Comment:
The order of the first types must follow the defautl for the enums, so `char` must not be here. See also the `NOTE`.

Reply:
Ok removed